mkosi/util.py

```diff
@@ -257,3 +257,29 @@ def current_home_dir() -> Optional[Path]:
 
 def unique(seq: Sequence[T]) -> list[T]:
     return list(dict.fromkeys(seq))
+
+
+def initrd_copy_host_config(staging_dir: str) -> list[str]:
```

Comment:
This is too generic for util.py, you can just keep it in initrd.py and import it from addon.py